src/main/java/kg/jarkyn/server/Response/ResponseStatusLine.java

```diff
@@ -0,0 +1,28 @@
+package kg.jarkyn.server.Response;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ResponseStatusLine {
+    private static final String PROTOCOL = "HTTP/1.1";
+    private final String statusCode;
+    private final static Map<String, String> STATUS_DESCRIPTIONS = new HashMap<String, String>() {
```

Comment:
200 will always be OK,
404 will always be NOT FOUND.
This HashMap feels like it would like to be a full grown Enum :)